CMakeLists.txt

```diff
@@ -2219,29 +2217,49 @@ if(FAAD)
   endif()
 endif()
 
+# FDK-AAC is loaded dynamically at runtime by EncoderFdkAac using QLibrary,
+# so copy it into the Windows and macOS packages, but do not link to it.
+find_library(FDK_AAC_LIBRARY fdk-aac)
+# On Windows find_library finds the .lib file, but the installer needs the .dll file.
+find_file(FDK_AAC_DLL fdk-aac.dll PATH_SUFFIXES ${CMAKE_INSTALL_BINDIR})
+if(APPLE AND MACOS_BUNDLE AND FDK_AAC_LIBRARY)
+  message(STATUS "Found fdk-aac: ${FDK_AAC_LIBRARY}")
+  file(COPY ${FDK_AAC_LIBRARY}
+    DESTINATION ${MIXXX_INSTALL_PREFIX}/Contents/Frameworks
+    FOLLOW_SYMLINK_CHAIN
+  )
+elseif(WIN32 AND FDK_AAC_DLL)
+  message(STATUS "Found fdk-aac DLL: ${FDK_AAC_DLL}")
+  install(FILES ${FDK_AAC_DLL} DESTINATION ${MIXXX_INSTALL_BINDIR})
+elseif(FDK_AAC_LIBRARY)
+  message(STATUS "Found fdk-aac: ${FDK_AAC_LIBRARY}")
+else()
+  message(STATUS "Could NOT find fdk-aac")
+endif()
+
 # FFmpeg 4.x support
 # FFmpeg is multimedia library that can be found http://ffmpeg.org/
 find_package(FFMPEG COMPONENTS libavcodec libavformat libavutil libswresample)
-option(FFMPEG "FFmpeg 4.x support" OFF)
+cmake_dependent_option(FFMPEG "FFmpeg 4.x support" ON "FFMPEG_FOUND" OFF)
```

Comment:
Please revert this. It will fail if the FFMPEG version is too old like on Ubuntu Bionic.

Reply:
I explicitly disabled FFMPEG on Ubuntu 18.04 for this reason.

Reply:
This does not help for home builds. The old version was just fine.

Reply:
Is anyone actually building on Ubuntu 18.04...

Reply:
If they are, they will get an error message informing them the version of FFMPEG is too old so they should be able to figure out to disable FFMPEG.

Reply:
For 2.3 it's still an official supported platform - it must compile out of the box on any supported platform.

Reply:
FFmpeg is now off by default if it is found but too old a67122042b26aebf33b810e05d27e4e21192f022